content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -328,8 +268,6 @@ Closing connection
 ```
  Important note
 Port number may cause an error if the port is already in use. To fix this, change the port number to a value that is not commercially owned.
-
 ### Conclusion 
-
-In this article,we got an insight into sockets and TCP/IP protocol. We went through what socket programming entails and its relation to Java. We have also seen a clear picture of client/server relationship, including the flow of data between the two.
-I urge the reader to use the knowledge learned above to write networking programs in java programming language.
+In this article, we got an insight into sockets and TCP/IP protocol. We went through what socket programming entails and its relation to Java. We have also seen a clear picture of the client/server relationship, including the flow of data between the two.
+I urge the reader to use the knowledge learned above to write networking programs in a java programming language.
```

Comment:
Change: 

> a java programming language.

To: 

> the java programming language

It doesn’t make sense to say *a Java programming language*, since there is only one. Instead, *the* is the proper word choice.